containers/docker-compose.yaml

```diff
@@ -441,10 +441,15 @@ services:
       - SIAS_CONSOLE_LOG=true
       - SIAS_AUTH_USERNAME=cathyadmin
       - SIAS_AUTH_PASSWORD=supersecretsquirrel
+      - SECTIGO_USERNAME=cathyadmin
+      - SECTIGO_PASSWORD=supersecretsquirrel
+      - SECTIGO_PROFILE=17
+      - SECTIGO_ENVIRONMENT=testing
```

Comment:
Just double-checking, should this be testing or staging?

Reply:
Docker compose is for local testing, so I think it should be testing. The staging configuration is in the devops repo.